src/components/MoneyRequestReportView/MoneyRequestReportTransactionList.tsx

```diff
@@ -124,28 +124,29 @@ function MoneyRequestReportTransactionList({report, transactions, reportActions,
 
     const {sortBy, sortOrder} = sortConfig;
 
-    const newTransactionID = useMemo(() => {
+    const newTransactionsID = useMemo(() => {
         if (!prevTransactions || transactions.length === prevTransactions.length) {
             return CONST.EMPTY_ARRAY as unknown as string[];
         }
 
         return transactions
             .filter((transaction) => !prevTransactions.some((prevTransaction) => prevTransaction.transactionID === transaction.transactionID))
-            .reduce((latest, t) => {
-                const inserted = t?.inserted ?? 0;
-                const latestInserted = latest?.inserted ?? 0;
-                return inserted > latestInserted ? t : latest;
-            }, transactions.at(0))?.transactionID;
+            .reduce((acc, t) => {
+                if (!prevTransactions.some((prevTransaction) => prevTransaction.transactionID === t.transactionID)) {
+                    acc.push(t.transactionID);
+                }
+                return acc;
```

Comment:
This `reduce` seems redundant with the `filter` above. Can we simplify this? Maybe something like:

```
return transactions
        .filter((transaction) =>
            !prevTransactions.some((prev) => prev.transactionID === transaction.transactionID)
        )
        .map((t) => t.transactionID);
```

Reply:
Good catch! I forgot to remove that filter 🤦